absl/strings/internal/str_format/checker.h

```diff
@@ -22,7 +22,7 @@
 // Compile time check support for entry points.
 
 #ifndef ABSL_INTERNAL_ENABLE_FORMAT_CHECKER
-#if ABSL_HAVE_ATTRIBUTE(enable_if) && !defined(__native_client__)
+#if ABSL_HAVE_ATTRIBUTE(enable_if) && !defined(__native_client__) && !defined(__INTELLISENSE__)
 #define ABSL_INTERNAL_ENABLE_FORMAT_CHECKER 1
 #endif  // ABSL_HAVE_ATTRIBUTE(enable_if) && !defined(__native_client__)
```

Comment:
Fix this comment since it now doesn't match.

Reply:
the new comment will cross 80-char limit. How do you suggest to format it @derekmauro ?

Reply:
The way you wrote it is fine.